packages/@react-aria/landmark/test/useLandmark.test.tsx

```diff
@@ -1116,6 +1120,40 @@ describe('LandmarkManager', function () {
         </Main>
       </div>
     );
-    expect(spyWarn).toHaveBeenCalledWith('Page contains more than one landmark with the \'navigation\' role and \'nav label 1\' label. If two or more landmarks on a page share the same role, they must have unique labels.');
+    expect(spyWarn).toHaveBeenCalledWith('Page contains more than one landmark with the \'navigation\' role and \'nav label 1\' label. If two or more landmarks on a page share the same role, they must have unique labels: ', navs);
+  });
+
+  it('focus restores to previously focused landmark after blur and F6.', function () {
+    let {getByRole} = render(
+      <div>
+        <Navigation>
+          <ul>
+            <li><a href="/home">Home</a></li>
+            <li><a href="/about">About</a></li>
+            <li><a href="/contact">Contact</a></li>
+          </ul>
+        </Navigation>
+        <Main>
+          <TextField label="First Name" />
+        </Main>
+      </div>
+    );
+    let nav = getByRole('navigation');
+
+    fireEvent.keyDown(document.activeElement, {key: 'F6'});
+    fireEvent.keyUp(document.activeElement, {key: 'F6'});
+    expect(nav).toHaveAttribute('tabIndex', '-1');
+    expect(document.activeElement).toBe(nav);
+
+    act(() => {
+      (document.activeElement as HTMLElement).blur();
+    });
```

Comment:
Using fireEvent with mousedown/up on the document.body won't actually blur this or move focus to the document.body because technically the body isn't focusable in the traditional sense and fireevent wont' move focus to it because of that